cmd/node/config/config.toml

```diff
@@ -12,13 +12,13 @@
    StartInEpochEnabled = true
 
    # DisableDeploy represents the flag to enable/disable deployment of smart contracts
-   DisableDeploy = false
```

Comment:
Restore all the modified flags in this file

Reply:
done